controller/eks-cluster-config-handler.go

```diff
@@ -246,6 +251,18 @@ func deleteNodeGroups(clusterName string, eksService *eks.EKS, nodeGroups []v13.
 }
 
 func (h *Handler) checkAndUpdate(config *v13.EKSClusterConfig, eksService *eks.EKS, svc *cloudformation.CloudFormation) (*v13.EKSClusterConfig, error) {
+	if err := validateUpdate(config); err != nil {
+		// validation failed, will be considered a failing update until resolved
+		config = config.DeepCopy()
+		config.Status.Phase = eksConfigUpdatingPhase
+		var updateErr error
+		config, updateErr = h.eksCC.UpdateStatus(config)
+		if updateErr != nil {
+			return config, updateErr
+		}
```

Comment:
nit:
```suggestion
		if config, updateErr := h.eksCC.UpdateStatus(config); updateErr != nil {
			return config, updateErr
		}
```

Reply:
I suppose I could still do this but I purposely did not because I want configs value to outlive this scope and usually use this syntax to contain.